### PR TITLE
network: DHCP lease load SIP copy paste error

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: src/libsystemd-network/sd-dhcp-lease.c [hunk 1]
```diff
@@ -1175,7 +1175,7 @@ int dhcp_lease_load(sd_dhcp_lease **ret, const char *lease_file) {
                 if (r < 0)
                         log_debug_errno(r, "Failed to deserialize SIP servers %s, ignoring: %m", sip);
                 else
-                        lease->ntp_size = r;
+                        lease->sip_size = r;
```
Comment (yuwata) [inline src/libsystemd-network/sd-dhcp-lease.c:1178]: Ouch!

**Comment by yuwata** [comment]
Looks good for me, but CIs do not like this.

**Comment by ssahani** [comment]
Thanks for the review @yuwata. Should pass now.

**Comment by keszybz** [comment]
Looks nice.

**Pull request merged**